packages/infra/client/container-runner/src/container.rs

```diff
@@ -32,8 +32,8 @@ pub fn run(
 		.context("empty `actor_path`")?
 		.to_string_lossy()
 		.to_string();
-	let oci_bundle_path = actor_path.join("oci-bundle");
-	let oci_bundle_config_json = oci_bundle_path.join("config.json");
+	let fs_path = actor_path.join("fs");
```

Comment:
nit: technically the fs root is in `oci-bundle/root`, so naming this "fs" is a bit confusing for me.

thoughts on moving it to `fs/oci-bundle`? also gives us flexibility to add more files in the future.

Reply:
Its sort of a double embedding, i didnt bother trying to make a custom fs for the bundle's root folder and instead put it inside the `fs` folder. so the `fs` folder is in fact a file system, there just happens to be an oci fs inside

Reply:
roger, i misunderstood this. i thought that we were using this as a "staging folder" for all the untar/umoci/skopeo operations. looks like it's just for running. disregard.